transforms/move-pod-description-to-content/README.md

```diff
@@ -0,0 +1,61 @@
+# move-pod-description-to-content
+
+The `description` prop of the Pod component has been deprecated in favour of placing a customized Typography Component inside the Pod Content. This codemod removes all `description` prop instances and places a `Typography` inside the Pod Content or inside the Accordion content in case of collapsible Pod.
```

Comment:
Sometimes the component names are in code but sometimes not.

e.g. `Typography` vs Typography.

Reply:
changed all component references